XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1082,7 +1082,7 @@
       <Version>6.1.9</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.2.190611001-prerelease</Version>
+      <Version>2.2.190822001-prerelease</Version>

```

Comment:
Is this correct? 2.2 is released now, right?

Reply:
It is released, and doesn't include any pre-release features. So when I updated to release version, the ScrollViewer (preview) page loudly complained.